### ISSUE TITLE
Bloks does not use the same text boundary as the Redliner tool. Please use the same math as Redliner for calculating the text bounds since its bounds are correct.

### ISSUE THREAD

**Issue by Jeremy-Knudsen** [WestonThayer/Bloks#16]

**Comment by Jeremy-Knudsen** [comment]
If I set the text to be area text and then, in the area text options dialog, set "First Baseline" to "Legacy" and check "Auto Size" then I can get better vertical alignment results.

**Comment by Jeremy-Knudsen** [comment]
If I know the type size is not going to change size (e.g. it always will be 15pt) then I can change the text to area text and then, in the area text options dialog set "First Baseline" to "Fixed" and adjust the pixel value to a value that aligns the text to what I need it to (usually 18px for Segoe UI Regular 15pt)

**Comment by WestonThayer** [comment]
Yep, area text has a bunch of options that are helpful for configuring the size. I'll have to think about this one a bit. It's hard because that Redliner tool is to match XAML's specific text rendering engine. Text does not have the same box height on different platforms (Android, iOS). We'd have to create different rendering modes or something for text... it gets sticky. Do you happen to know if that Redliner tool matches browsers?
